application/controllers/admin/Statistics.php

```diff
@@ -685,13 +685,24 @@ public function simpleStatistics($surveyid)
         $rows = Question::model()->primary()->getQuestionList($surveyid);
         ;
 
+        $rawQuestions = Question::model()->findAll("sid = :sid and `type` = :type", [
```

Comment:
as those are only questions of type COLON_ARRAY_NUMBERS, I would include that in the variable names